workflows/rnaseq/rnaseq-standard-fastq.wdl

```diff
@@ -35,7 +35,7 @@ import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/ngs
 import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/htseq.wdl"
 import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/samtools.wdl"
 import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/deeptools.wdl"
-import "https://raw.githubusercontent.com/stjude/xenocp/master/wdl/workflows/xenocp.wdl" as xenocp_workflow
+import "https://raw.githubusercontent.com/stjude/XenoCP/3.1.4/wdl/workflows/xenocp.wdl" as xenocp_workflow
```

Comment:
Could you just explicitly comment on why this is needed?

Reply:
I'll let @a-frantz make the update, but we decided that we should depend on an explicit version for externally maintained tools. This was prompted by changes to SEAseq, where a dependency was renamed ("util.wdl" -> "seaseq_util.wdl") in their repository which left our main branch implementation broken.

Reply:
XenoCP falls into a gray area... It's maintained by us, but is also an external repo. With SEAseq changes causing our CI to fail, me and @adthrasher decided external repos should point to a tagged release for improved stability. "External repo" would include XenoCP.

Reply:
I think it's fine to include the explicit version then. I would consider this an external repo.